packages/design-tokens/docs/pages/what-are-design-tokens.stories.mdx

```diff
@@ -0,0 +1,14 @@
+import { Meta } from '@storybook/addon-docs/blocks'
+
+<Meta title="Design Tokens/What are design tokens" parameters={{ docsOnly: true }} />
+
+
+# What are Kaizen design tokens?
+Kaizen's design tokens are a hierarchical set of variables that partly define what our design system *is*. They contain definitions of design system constants related (but not limited) to color, typography, spacing and layout,
+but don't define exactly what components should look like, otherwise they would be containing or re-inventing HTML and CSS. 
+They are the set of variables that accompany Kaizen components to make them look and feel as consistent as possible, and are available to be consumed in your own compositions.
+
+
+As of V3, design tokens are also **themable**; they are intended to be used as if they can be **switched at runtime**. Because of this, we employ [CSS Variables](https://www.w3schools.com/css/css3_variables.asp)
+to support this feature, so when consuming tokens in SASS/LESS you should assume they will contain values such as `var(--kz-var-color-...)`, rather than concrete values such as `1.5rem` or `#fff`.
+
```

Comment:
I don't think we need this page.

I'm tempted to stick to "How Tos" in Storybook, and keep explanations elsewhere.

The README for design-tokens has a pretty good explanation of the concept (but will need updating). I think I'd prefer to keep this explanation there

https://github.com/cultureamp/kaizen-design-system/tree/master/packages/design-tokens#about